.travis.yml

```diff
@@ -1,19 +1,20 @@
 language: php
 
+dist: bionic
+
 php:
   - 7.1
+  - 7.2
+  - 7.3
+  - 7.4
 
 env:
   matrix:
     - COMPOSER_FLAGS="--prefer-lowest"
     - COMPOSER_FLAGS=""
 
-before_script:
-  - travis_retry composer self-update
-  - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-source
+install:
+  - travis_retry composer update ${COMPOSER_FLAGS}
 
 script:
   - vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover
-
-after_script:
-  - php vendor/bin/ocular code-coverage:upload --format=php-clover coverage.clover
```

Comment:
Looks like ocular was never installed. eg see the last build on master: https://travis-ci.org/spatie/async/jobs/629063019

<img width="1087" alt="image" src="https://user-images.githubusercontent.com/2829600/71410842-c4cfc500-263e-11ea-8a10-32a4ad9c9961.png">